bridge:
  topics:
    cmd_sub: "/turtlebot3_bridge/command"
turtlebot3:
  rate_hz: 10
  circle:
    velocities:
      x: 0.15
      z: 0.3
    thresholds:
      angular_rad: 0.03
  polygon:
    velocities:
      x: 0.2
      z: 0.2
    edge:
      length_meter: 0.4
    thresholds:
      angular_rad: 0.02
  topics:
    cmd_pub: "/cmd_vel"
    odom_sub: "/odom"
